kalite/testing/behave_helpers.py

```diff
@@ -201,14 +201,11 @@ def _find_elem_with_wait(context, by, wait_time=MAX_WAIT_TIME):
     context: a behave context
     by: A tuple selector used by Selenium
     wait_time: The max time to wait in seconds
-    Returns the element if found or None
+    Returns the element if found or raises TimeoutException
     """
-    try:
-        return WebDriverWait(context.browser, wait_time).until(
-            EC.presence_of_element_located(by)
-        )
-    except TimeoutException:
-        return None
+    return WebDriverWait(context.browser, wait_time).until(
```

Comment:
Oh, I see. Disregard my earlier comment. This is probably the better way to do it, anyway.